packages/test-framework/src/fixture/pageObject/Assertions.ts

```diff
@@ -27,4 +27,14 @@ export class Assertions {
             /Your order number is \d*/,
         );
     }
+
+    async shouldSeeErrorModal(): Promise<void> {
+        const page = this.page;
+
+        await page.locator('data-test=modal-body').waitFor({ state: 'visible' });
+        await expect(page.locator('data-test=modal-body')).toBeVisible();
+        await expect(page.locator('#errorModalMessage')).toHaveText(
+            'The shipping price you were quoted is no longer valid. Click OK to see the most up-to-date shipping prices.',
+        );
```

Comment:
Please make the text as a parameter so that this method can be applied to different error cases.